docs/source/pillars/computing_technology.md

```diff
@@ -2,55 +2,55 @@
 
 # Computing technology
 
-This capability concerns what the TRE operator does to manage systems for storing, retrieving, analysing and sending information.
-
 ```{figure} ../../images/Capability_Map/full.drawio.svg
 :alt: SATRE Pillars Capability Map
 :align: center
 
 SATRE Pillars Capability Map
 ```
 
+This pillar concerns actions taken by the TRE operator to manage systems for storing, retrieving, analysing and sending information.
+
+Each TRE operator will have its own computing technology requirements.
+Requirements are drawn from information governance requirements which primarily specify the controls needed within the technical and computing infrastructure.
+Researcher personas influence the according to the technical knowledge and experience of researchers along with the work they need to perform within the system, a data scientist will have very different requirements to a clinician for example.
+The required compute resources will vary according to the scale of data and computational techniques employed during research.
+
 <!-- See all pillars of the SATRE Pillars Capability Map here: {ref}`satre_pillars` -->
 
 ## End user computing
 
 _The ability of the TRE operator to provide and manage devices, workspaces, interfaces and applications used by researchers to interact with underlying systems and data._
 
-### User interface
+### End user computing interfaces
 
-_The interfaces used for interacting with the TRE management system and the TRE workspace._
+Software or system that allows people to interact with computers, applications, and services.
+This application group includes desktop, command line and code submission interface.
 
 ```{list-table}
 :header-rows: 1
-:name: tab-end-user-user-interface
+:name: tab-end-user-computing-interfaces
 
 * - Statement
   - Guidance
   - Importance
-* - Your TRE should be accessed via a user interface accessible using commonly available applications.
-  - TREs which allow users to connect from their own devices should not require the installation of any bespoke TRE application on the user's device.
-    In practice a web browser is the most common way to achieve this.
+
+* - You should disable the ability to copy data out of your TRE via the system clipboard.
+  - A TRE user must not be able to copy sensitive data out of a workspace using the system clipboard.
+    A TRE may allow user to paste text into a workspace.
   - Recommended
```

Comment:
Feels like a "Mandatory" to me. What do you think?

Reply:
I would say so. It feels like it is a big risk for not much benefit. Maybe TREs who are much more confident about safe people are less concerned here. It might be that we came to recommended here to not be too prescriptive.

In any case, I don't think we can say "should not" in the statement and then "must not" in the guidance.

@manics do you remember any extra background context here?